src/Engine/Manager/ParticleManager.cpp

```diff
@@ -35,10 +35,10 @@ void ParticleManager::Update(World& world, float time, bool preview) {
     }
 }
 
-void ParticleManager::RenderParticleSystem(const glm::mat4& viewProjectionMatrix) {
+void ParticleManager::RenderParticleSystem(const glm::mat4& viewMatrix, const glm::mat4&  projectionMatrix) {
 
     for (Component::ParticleSystemComponent* comp : particleSystems.GetAll()) {
-        particleSystemRenderers[comp]->Draw(textureAtlas, textureAtlasRowNumber, viewProjectionMatrix, emitterSettings[comp]);
+        particleSystemRenderers[comp]->Draw(textureAtlas, textureAtlasRowNumber, viewMatrix, projectionMatrix, emitterSettings[comp]);
```

Comment:
Either put it all on one line and remove brackes on the for loop or split the function up like this:
function(param,
param, 
param)